.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  presets: ['react', 'es2015', 'stage-0']
+  presets: ['react', 'env', 'stage-0']
```

Comment:
I think Babel-Preset-Env usually requires or at least recommends a modicum of configuration… I don't recall all the defaults offhand.

Also I am personally not a fan of `stage-0` ever. I feel that `stage-2` is a [much better threshold](https://tc39.github.io/process-document/) for inclusion in a project.

> Stage 1 Spec Quality: none.

> Stage 2 Spec Quality: Draft: all **major** semantics, syntax and API are covered, but TODOs, placeholders and editorial issues are expected.

I could make an argument that even stage 3 or 4 are preferable, but I get the fun of using new features "early."

Reply:
[Yep](https://github.com/babel/babel-preset-env/blob/master/README.md#babel-preset-env----):

> Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest

You can see an example config that works for Bones, with associated conversation, in [this PR](https://github.com/FullstackAcademy/bones/pull/106/files).